src/wombats/game/decisions/smoke.clj

```diff
@@ -0,0 +1,89 @@
+(ns wombats.game.decisions.smoke
+  (:require [wombats.game.decisions.helpers :as dh]
+            [wombats.game.utils :as gu]
+            [wombats.arena.utils :as au]))
+
+(defn- get-smoked-coords
+  "Return a collection of coords that should be smokescreened given a central starting point"
+  [coords arena-dims]
+  (conj (map #(gu/adjust-coords coords % arena-dims)
+             (range 8))
+        coords))
+
+(defn- calculate-throw-direction
+  "determines the direction smoke should be thrown based off of the
+  decision makers orientation and the specified throw direction"
```

Comment:
Going to keep for now, but will do an app wide find and replace when we settle on a good name